graph/layout/eades_test.go

```diff
@@ -0,0 +1,205 @@
+// Copyright ©2019 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package layout
+
+import (
+	"bytes"
+	"os/exec"
+	"path/filepath"
+	"reflect"
+	"testing"
+
+	"gonum.org/v1/gonum/graph/encoding/dot"
```

Comment:
regroup with the other `gonum/graph` imports?

Reply:
Done